packages/theme/src/theme.ts

```diff
@@ -1,69 +1,84 @@
 import merge from 'ts-deepmerge';
-import {borderRadius, borderWidth, iconSizes, spacings} from './sizes';
 import {ConfigurationOverride, overrideConfig} from './utils/override-config';
 import {
   AtBThemes,
-  NfkThemes,
-  FRAMThemes,
-  TromsThemes,
-  InnlandetThemes,
+  // NfkThemes,
+  // FRAMThemes,
+  // TromsThemes,
+  // InnlandetThemes,
 } from './themes';
-export type Themes = {
-  light: Theme;
-  dark: Theme;
+
+import {
+  AtBThemesFs,
+  // NfkThemes,
+  // FRAMThemes,
+  // TromsThemes,
+  // InnlandetThemes,
+} from './themes-fs';
+
+export type Themes<T = Theme> = {
+  light: T;
+  dark: T;
 };
 export type Mode = keyof Themes;
 
 export type TextColor = 'primary' | 'secondary' | 'disabled';
 
 export type TextColorType = 'dark' | 'light';
 
+export type RadiusSizes = keyof Theme['border']['radius']
+
 export type ContrastColor = {
   background: string;
   text: string;
 };
-export type TransportColor = {
-  primary: ContrastColor;
-  secondary: ContrastColor;
+export type TransportColor<T = ContrastColor> = {
```

Comment:
At first glance I don't see why the generics are added. Are there any functionality based on it, or just to ensure that all fields uses the same type?

Reply:
We now expose both an old and a new structure. The new structure changes the `contrastColor` to 

```
export type ContrastColorFs = {
  background: string;
  foreground: {
    primary: string;
    secondary: string;
    disabled: string;
  };
};
```

To avoid having to repeat all Transport, Status, Interactive, ... color type definitions, we leverage generics to inject the ColorContrast type. Since this defaults to the old ContrastColor, no breaking changes are introduced here